lib/Backend/BackendUtils.cpp

```diff
@@ -144,15 +241,24 @@ runtime::RuntimeBundle runtime::RuntimeBundle::create(const Function &F) {
   }
 
   // Allocate placeholders.
-  for (auto const *V : F.findPlaceholders()) {
+  // Placeholders should be allocated in a order of Input|InputOutput|Output.
+  std::vector<std::tuple<const Placeholder *, bool, bool>>
+      contiguousPlaceholders =
+          glow::getContiguousPlaceHolder<Function, std::list<Placeholder *>>(
```

Comment:
Is it necessary to explicitly provide the type parameters for the template function? Is it because the compiler cannot deduce them automatically?